simulators/portal-mesh/src/main.rs

```diff
@@ -178,3 +190,54 @@ dyn_async! {
         }
     }
 }
+
+dyn_async! {
+    async fn test_find_nodes_distance_of_client_c<'a>(clients: Vec<Client>) {
+        let (client_a, client_b, client_c) = match clients.iter().collect_tuple() {
+            Some((client_a, client_b, client_c)) => (client_a, client_b, client_c),
+            None => {
+                panic!("Unable to get expected amount of clients from NClientTestSpec");
+            }
+        };
+
+        let target_enr = match client_b.rpc.node_info().await {
+            Ok(node_info) => node_info.enr,
+            Err(err) => {
+                panic!("Error getting node info: {err:?}");
+            }
+        };
+
+        // We are adding client C to our list so we then can assume only one client per bucket
+        let client_c_enr = match client_c.rpc.node_info().await {
+            Ok(node_info) => node_info.enr,
+            Err(err) => {
+                panic!("Error getting node info: {err:?}");
+            }
+        };
+
+        // seed enr into routing table
```

Comment:
note: `add_enr` does have to set the added node as "verified". Which I just realised I sorta promised to add into the specifications.

It should work for Fluffy nodes as it is what we already do.

Reply:
How are we standardizing the connection of nodes then, cause if that happened I will need to update most tests I would think